test/integration/multiplexed_integration_test.cc

```diff
@@ -1000,6 +1000,175 @@ name: encode-headers-return-stop-all-filter
   EXPECT_EQ(count * size + added_decoded_data_size * 2, response->body().size());
 }
 
+class MetadataIntegrationTest : public HttpProtocolIntegrationTest {
```

Comment:
can we change Http2MetadataIntegrationTest to be MetadataIntegrationTest and run all the tests for H2/H3?

Reply:
Done!